SPECS/orangefs/orangefs.spec

```
@@ -1,10 +1,20 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
-Name: orangefs
-Version: 2.9.8
-Release: 1%{?dist}
-Summary: Parallel network file system client
-URL: https://www.orangefs.org/
+%global _hardened_build 1
```

Not needed - we build this way by default.
```suggestion
%global _hardened_build 1
```
```suggestion
```